tests/robottelo/test_decorators.py

```diff
@@ -75,6 +96,78 @@ def bomb(_):
         decorators._get_bugzilla_bug = bomb
         self.assertFalse(decorators.bz_bug_is_open(self.bug_id))
 
+    def test_upstream_with_whiteboard(self):
+        """Assert that upstream bug is not affected by whiteboard texts"""
+        class MockBug(object):  # pylint:disable=R0903
+            """A mock bug"""
+            status = None
+            whiteboard = None
+        conf.properties['main.upstream'] = str(1)  # upstream mode
```

Comment:
Any reason not to write this as `'1'`? Same question applies throughout this file.